[EDU-3284] refactor: add disclaimer grafana cloud (#497)
aziontech/docs: src/content/docs/en/pages/guides/grafana/grafana-cloud/index.mdx

permalink: /documentation/products/guides/graphql-grafana-cloud/
---

:::caution[warning]

> The Grafana plugin described in this guide **isn't** created or managed by Azion. It's a possible tool for users who wish to use the Grafana Cloud platform.

Reviewer: ```suggestion
The Grafana plugin described in this guide **isn't** created or managed by Azion. It's an available tool for users who want to use the Grafana Cloud platform.
```
